common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -485,15 +484,42 @@ def setup_thread(self):
         thread_fixture.push()
         return thread
 
-    def test_discussion_on_team_page(self):
+    def test_discussion_on_my_team_page(self):
         """
-        Scenario: Team Page renders a team discussion.
+        Scenario: Team Page renders a discussion for a team to which I belong.
         Given I am enrolled in a course with a team configuration, a topic,
-            and a team belonging to that topic
-        When I post a thread in the team's discussion
+            and a team belonging to that topic of which I am a member
+        When the team has a discussion with a thread
         And I visit the Team page for that team
         Then I should see a discussion with the correct discussion_id
         And I should see the thread which I had posted
+        And I should see controls to change the state of the discussion
+        """
+        thread = self.setup_thread()
+        self.create_membership(self.user_info['username'], self.team['id'])
+        self.team_page.visit()
+        self.assertEqual(self.team_page.discussion_id, self.team['discussion_topic_id'])
+        discussion = self.team_page.discussion_page
+        self.assertTrue(discussion.is_browser_on_page())
+        self.assertTrue(discussion.is_discussion_expanded())
+        self.assertEqual(discussion.get_num_displayed_threads(), 1)
+        self.assertTrue(discussion.has_thread(thread['id']))
+        self.assertTrue(discussion.q(css='.post-header-actions').present)
+        self.assertTrue(discussion.q(css='.add-response').present)
+        self.assertTrue(discussion.q(css='.new-post-btn').present)
+
+    def test_discussion_on_other_team_page(self):
+        """
```

Comment:
I'm also going to want a test case that checks that a "privileged user" (say, Discussion TA) has enabled controls, even if she is not in the team AND is not course staff.

I'd recommend creating a helper method that can do all the necessary checks based on the expected enabled state.